symbolic-symcache/src/transform/il2cpp.rs

```diff
@@ -24,9 +22,8 @@ impl Transformer for LineMapping {
         // TODO: this allocates, which is especially expensive since we run this transformer for
         // every single source location (without dedupe-ing files). It might be worth caching this
         let full_path = full_path(&sl.file);
-        if let Some((mapped_file, mapped_line)) = self.lookup(&full_path, sl.line) {
+        if let Some((mapped_file, mapped_line)) = self.lookup(&*full_path, sl.line) {
```

Comment:
`full_path` should be a `String`, not sure what `&*` would even do in that case?